listener/Dockerfile

```diff
@@ -110,20 +117,125 @@ RUN mkdir -p build \
   && cmake .. -DUNTRUSTED_ONLY=1 \
   && make
 
+
+
+# -------------=== common/python build ===-------------
+
+#Build common/cpp intermediate docker image
+FROM python_image as common_python_image
+
+RUN apt-get update \
+ && apt-get install -y -q \
+    make
+
+# Copy openssl build artifacts from openssl_image
+COPY --from=openssl_image /usr/local/ssl /usr/local/ssl
+COPY --from=openssl_image /usr/local/bin /usr/local/bin
+COPY --from=openssl_image /usr/local/include /usr/local/include
+COPY --from=openssl_image /usr/local/lib /usr/local/lib
+
+RUN ldconfig \
+ && ln -s /etc/ssl/certs/* /usr/local/ssl/certs/
+
+COPY VERSION /project/avalon/
+COPY ./bin /project/avalon/bin
+
+ENV TCF_HOME=/project/avalon
+
+COPY ./common/python /project/avalon/common/python
+
 WORKDIR /project/avalon/common/python
 
 RUN echo "Building Avalon Common Python\n" \
- && make 
+ && make
+
+
+
+
+

```

Comment:
Why is there 5 blank lines here (and below starting at line 218)?

Reply:
removed